[ES] Add `History of osu!/2017` (#9222)
ppy/osu-wiki: wiki/History_of_osu!/2017/es.md

# Historia de osu! 2017

## Enero

> [Ephemeral](https://osu.ppy.sh/users/102335) publicó una noticia sobre el nuevo año para asegurar que sucederán muchas cosas importantes en 2017. [S3RL](https://osu.ppy.sh/beatmaps/artists/9) y [nanobii](https://osu.ppy.sh/beatmaps/artists/10) fueron anunciados como los nuevos artistas destacados de osu!. [cYsmix](https://osu.ppy.sh/beatmaps/artists/2), [yuki.](https://osu.ppy.sh/beatmaps/artists/4) y [Helblinde](https://osu.ppy.sh/beatmaps/artists/5) cada uno lanzó un nuevo álbum dando a osu! 26 nuevas canciones para mapear. Aspire regresa para 2017.

Reviewer: ```suggestion
[Ephemeral](https://osu.ppy.sh/users/102335) publicó una noticia sobre el nuevo año para asegurar que sucederán muchas cosas importantes en 2017. [S3RL](https://osu.ppy.sh/beatmaps/artists/9) y [nanobii](https://osu.ppy.sh/beatmaps/artists/10) fueron anunciados como nuevos artistas destacados de osu!. [cYsmix](https://osu.ppy.sh/beatmaps/artists/2), [yuki.](https://osu.ppy.sh/beatmaps/artists/4) y [Helblinde](https://osu.ppy.sh/beatmaps/artists/5) lanzaron cada uno un nuevo álbum dando a osu! 26 nuevas canciones para beatmapear. Aspire regresa para 2017.
```

Author: Está raro decir "beatmapear", la gran mayoría dice "mapear" al momento de querer crear un mapa. 

No he escuchado a nadie decir "voy a beatmapear esta canción". Lo otro me parece bien.

```suggestion
[Ephemeral](https://osu.ppy.sh/users/102335) publicó una noticia sobre el nuevo año para asegurar que sucederán muchas cosas importantes en 2017. [S3RL](https://osu.ppy.sh/beatmaps/artists/9) y [nanobii](https://osu.ppy.sh/beatmaps/artists/10) fueron anunciados como nuevos artistas destacados de osu!. [cYsmix](https://osu.ppy.sh/beatmaps/artists/2), [yuki.](https://osu.ppy.sh/beatmaps/artists/4) y [Helblinde](https://osu.ppy.sh/beatmaps/artists/5) lanzaron cada uno un nuevo álbum dando a osu! 26 nuevas canciones para mapear. Aspire regresa para 2017.
```

Links:

> - [¡Feliz año nuevo!](https://osu.ppy.sh/home/news/2017-01-01-happy-new-year)
> - [Nuevos artistas destacados: S3RL y nanobii](https://osu.ppy.sh/home/news/2017-01-09-new-featured-artists-s3rl-and-nanobii)
> - [Aspire regresa para 2017](https://osu.ppy.sh/home/news/2017-01-21-aspire-returns-for-2017)

Reviewer: ```suggestion
- [Happy New Year!](https://osu.ppy.sh/home/news/2017-01-01-happy-new-year)
- [New Featured Artists: S3RL and nanobii](https://osu.ppy.sh/home/news/2017-01-09-new-featured-artists-s3rl-and-nanobii)
- [Aspire Returns For 2017](https://osu.ppy.sh/home/news/2017-01-21-aspire-returns-for-2017)
```

## Febrero

[Ben Briggs](https://osu.ppy.sh/beatmaps/artists/11) fue anunciado como otro artista destacado. ¡La primera compilación de [lazer](/wiki/Client/Release_stream/Lazer) de osu! fue lanzada!

> Links:

Reviewer: ```suggestion
Enlaces:
```

> - [Nuevo artista destacado: Ben Briggs](https://osu.ppy.sh/home/news/2017-02-08-new-featured-artist-ben-briggs)
> - [Versión 0.0.3 · ppy/osu](https://github.com/ppy/osu/releases/tag/v0.0.3)

Reviewer: ```suggestion
- [New Featured Artist: Ben Briggs](https://osu.ppy.sh/home/news/2017-02-08-new-featured-artist-ben-briggs)
- [Release 0.0.3 · ppy/osu](https://github.com/ppy/osu/releases/tag/v0.0.3)
```

## Marzo

> Ha comenzado el Monthly Beatmapping Contest que gira en torno a osu!catch. [LukHash](https://osu.ppy.sh/beatmaps/artists/12) fue anunciado como otro artista destacado. La votación de Aspire 2017 ha comenzado. Se anunció que Destacados coexistiría con el Gráfico de clasificación. Se abrió el noveno concurso de fanart con temática de primavera.

Reviewer: ```suggestion
Ha comenzado el Monthly Beatmapping Contest que gira en torno a osu!catch. [LukHash](https://osu.ppy.sh/beatmaps/artists/12) fue anunciado como otro artista destacado. La votación de Aspire 2017 ha comenzado. Se anunció que Destacados coexistiría con el Gráfico de clasificación. Se abrió el noveno concurso de arte de fans con temática de primavera.
```

> Links:

Reviewer: ```suggestion
Enlaces:
```

- [Monthly Beatmapping Contest March 2017: osu!catch](https://osu.ppy.sh/home/news/2017-03-01-monthly-beatmapping-contest-march-2017-osucatch)
- [New Featured Artist: LukHash](https://osu.ppy.sh/home/news/2017-03-03-new-featured-artist-lukhash)
- [Aspire 2017 - Voting Now Open!](https://osu.ppy.sh/home/news/2017-03-13-aspire-2017-voting-now-open)
- [Introducing to you: Spotlights](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights)
- [Fanart Contest \#9: Spring](https://osu.ppy.sh/home/news/2017-03-21-fanart-contest-9-spring)

## Abril

> El Monthly Beatmapping Contest \#13 concluyó con [RyoKazuka](https://osu.ppy.sh/users/6258586) obteniendo el primer lugar. Apsire 2017: la etapa uno concluyó con [ProfessionalBox](https://osu.ppy.sh/users/3250792) ocupando el primer lugar. Se abrió la votación para el concurso de fanart de primavera. Se publicó el n.º 100(?) de osu!weekly. La osu!taiko World Cup 2017 concluyó con la victoria de Japón contra Taiwán. Antes de que terminara abril, los votos del concurso de fanart de primavera estaban listos, lo que permitió que [Evaine](https://osu.ppy.sh/users/7295733) obtuviera el primer lugar.

Reviewer: ```suggestion
El Monthly Beatmapping Contest \#13 concluyó con [RyoKazuka](https://osu.ppy.sh/users/6258586) obteniendo el primer lugar. Apsire 2017: la etapa uno concluyó con [ProfessionalBox](https://osu.ppy.sh/users/3250792) ocupando el primer lugar. Se abrió la votación para el concurso de arte de fans de primavera. Se publicó el n.º 100(?) de osu!weekly. La osu!taiko World Cup 2017 concluyó con la victoria de Japón contra Taiwán. Antes de que terminara abril, los votos del concurso de arte de fans de primavera estaban listos, lo que permitió que [Evaine](https://osu.ppy.sh/users/7295733) obtuviera el primer lugar.
```

Author: No creo que se refiera a un _problema_, sino a una edición

Reviewer: Pues prefiero fanart que fan art

Author: Ok

> Links:

Reviewer: ```suggestion
Enlaces:
```

- [Monthly Beatmapping Contest \#13 Results](https://osu.ppy.sh/home/news/2017-04-02-monthly-beatmapping-contest-13-results)
- [Aspire 2017 Stage One Concludes](https://osu.ppy.sh/home/news/2017-04-03-aspire-2017-stage-one-concludes)
- [Spring Fan Art Contest Voting Now Open](https://osu.ppy.sh/home/news/2017-04-11-spring-fan-art-contest-voting-now-open)
- [osu!weekly \#100](https://osu.ppy.sh/home/news/2017-04-15-osuweekly-100)
- [osu!taiko World Cup 2017 Results](https://osu.ppy.sh/home/news/2017-04-22-osutaiko-world-cup-2017-results)

## Mayo

> Se abrieron las inscripciones para la osu!catch World Cup 2017. [Kuba Oms](https://osu.ppy.sh/beatmaps/artists/13) apareció como otro artista de osu!. osu! anunció que estaría en la Anime Expo 2017. En casi más de un año, se anunció la mascota de osu!taiko. [Mocha](/wiki/Mascots#mocha) ha entrado en escena con [pippi](/wiki/Mascots#pippi), [Yuzu](/wiki/Mascots#yuzu), y [María](/wiki/Mascots#mani-&-mari). Por último, [Rin / Function Phantom](https://osu.ppy.sh/beatmaps/artists/14) fue anunciado como parte de los artistas destacados de osu!. Se abrió la votación para los concursos de mapeo de osu!mania y osu!catch.

Reviewer: ```suggestion
Se abrieron las inscripciones para la osu!catch World Cup 2017. [Kuba Oms](https://osu.ppy.sh/beatmaps/artists/13) apareció como otro artista de osu!. osu! anunció que estaría en la Anime Expo 2017. En casi más de un año, se anunció la mascota de osu!taiko. [Mocha](/wiki/Mascots#mocha) ha entrado en escena con [pippi](/wiki/Mascots#pippi), [Yuzu](/wiki/Mascots#yuzu), y [María](/wiki/Mascots#mani-&-mari). Por último, [Rin / Function Phantom](https://osu.ppy.sh/beatmaps/artists/14) fue anunciado como parte de los artistas destacados de osu!. Se abrió la votación para los concursos de mapping de osu!mania y osu!catch.
```

> Links:

Reviewer: ```suggestion
Enlaces:
```

- [osu!catch World Cup 2017 - Registrations now open!](https://osu.ppy.sh/home/news/2017-05-03-osucatch-world-cup-2017-registrations-now-open)
- [osu! at anime expo 2017](https://osu.ppy.sh/home/news/2017-05-09-osu-at-anime-expo-2017)
- [The new osu!taiko mascot is here!](https://osu.ppy.sh/home/news/2017-05-25-the-new-osutaiko-mascot-is-here)
- [New Featured Artist: Rin / Function Phantom](https://osu.ppy.sh/home/news/2017-05-30-new-featured-artist-rin-function-phantom)
- [Monthly Beatmapping Contest Voting Now Open: osu!catch and osu!mania](https://osu.ppy.sh/home/news/2017-06-01-monthly-beatmapping-contest-voting-now-open)

## Junio

> El equipo de osu! participa en la Anime Expo por los próximos 10 años de osu!. El 15 de junio de 2017, la imagen del menú principal/campo de juego se cambió a cuatro imágenes genéricas, pero nuevas. La imagen del campo de juego cambia cuando se cambia la canción actual. Se ha agregado a las [Opciones](/wiki/Client/Options) el soporte para habilitar o deshabilitar los fondos de temporada durante todo el año o verlos a veces. En algún momento alrededor de este mes, dentro del [servidor de Discord de osu!](/wiki/Community/osu!_Discord_server), se discutió que osu!weekly había muerto. A partir de este momento, cada publicación de noticias oficial y de la comunidad se publicará de forma independiente cuando esté lista, en lugar de esperar cada semana. Comenzó el concurso de fanart de Mocha en Verano.

Reviewer: ```suggestion
El osu! team participa en la Anime Expo por los próximos 10 años de osu!. El 15/6/2017, la imagen del menú principal/campo de juego se cambió a cuatro imágenes genéricas, pero nuevas. La imagen del campo de juego cambia cuando se cambia la canción actual. Se ha agregado a las [opciones](/wiki/Client/Options) el soporte para habilitar o deshabilitar los fondos de temporada durante todo el año o verlos a veces. En algún momento alrededor de este mes, dentro del [servidor de Discord osu!](/wiki/Community/osu!_Discord_server), se discutió que osu!weekly había muerto. A partir de este momento, cada publicación de noticias oficial y de la comunidad se publicará de forma independiente cuando esté lista, en lugar de esperar cada semana. Comenzó el concurso de arte de fans de Mocha en verano.
```

> El osu! team participa en la Anime Expo por los próximos 10 años de osu!. El 15 de junio de 2017, las imágenes del menú principal/campo de juego se cambiaron a cuatro imágenes genéricas, pero nuevas. La imagen del campo de juego cambia cuando se cambia la canción actual. Se ha agregado a las [opciones](/wiki/Client/Options) el soporte para habilitar o deshabilitar los fondos de temporada durante todo el año o verlos a veces. En algún momento alrededor de este mes, dentro del [servidor de Discord de osu!](/wiki/Community/osu!_Discord_server), se discutió que osu!weekly había muerto. A partir de este momento, cada publicación de noticias oficial y de la comunidad se publicará de forma independiente cuando esté lista, en lugar de esperar cada semana. Comenzó el concurso de fanart de Mocha en verano.

Reviewer: ```suggestion
El osu! team participa en la Anime Expo por los próximos 10 años de osu!. El 15 de junio de 2017, las imágenes del menú principal/campo de juego se cambiaron a cuatro imágenes genéricas, pero nuevas. La imagen del campo de juego cambia cuando se cambia la canción actual. Se ha agregado a las [opciones](/wiki/Client/Options) el soporte para habilitar o deshabilitar los fondos de temporada durante todo el año o verlos a veces. En algún momento alrededor de este mes, dentro del [servidor de Discord osu!](/wiki/Community/osu!_Discord_server), se discutió que osu!weekly había muerto. A partir de este momento, cada publicación de noticias oficial y de la comunidad se publicará de forma independiente cuando esté lista, en lugar de esperar cada semana. Comenzó el concurso de fanart de Mocha en verano.
```

Author: ¿No te parece que esté "de"? Yo lo veo bien

Reviewer: No, porque es el nombre

Author: Bueno, yo veo el nombre bien (osu!). Pero entiendo a lo que te refieres. 

Creo que es mejor especificar de quién es el Discord; por ejemplo, queda mejor decir "Me quiero unir al servidor de Discord de osu!" en lugar de decir "Me quiero unir al servidor de Discord osu!". Así se puede saber de qué servidor se está hablando.

Reviewer: No puede ser porque antes se llamaba `osu!dev` y no tenía sentido llamarlo `de osu!dev` así que ahora tampoco debería

Author: Pues sí, en ese caso no se vería bien. Diría que en este sí, creo que dependiendo del nombre del servidor se puede considerar usar una preposición o no.

Sigue sin convencerme del todo que no esté `de`, pero tampoco es que cambie todo el significado de lo que se quiere dar a entender.

> Links:

Reviewer: ```suggestion
Enlaces:
```

- [2017.609.0 – ppy blog](https://blog.ppy.sh/2017.609.0/)
- [2017.717.0 – ppy blog](https://blog.ppy.sh/2017.717.0/)
- [main menu background changes](https://osu.ppy.sh/community/forums/topics/606931)
- [Mocha in Summer Fanart Contest Begins!](https://osu.ppy.sh/home/news/2017-06-15-mocha-fanart-contest-now-open)

## Julio

> Se abren las inscripciones para la osu!mania 4K World Cup. Se abrió la votación para el concurso de fanart de Mocha en Verano. La osu!catch World Cup 2017 concluyó con Corea del Sur tomando el primer lugar de China. Se abrieron las inscripciones para el Community Mentorship Program Fall 2017. El último día de julio, el concurso de fanart de Mocha en verano concluyó con [2Su](https://osu.ppy.sh/users/6598966) obteniendo el primer lugar.

Reviewer: ```suggestion
Se abren las inscripciones para la osu!mania 4K World Cup. Se abrió la votación para el concurso de arte de fans de Mocha en Verano. La osu!catch World Cup 2017 concluyó con Corea del Sur tomando el primer lugar de China. Se abrieron las inscripciones para el Community Mentorship Program Fall 2017. El último día de julio, el concurso de arte de fans de Mocha en verano concluyó con [2Su](https://osu.ppy.sh/users/6598966) obteniendo el primer lugar.
```

> Links:

Reviewer: ```suggestion
Enlaces:
```

> - [osu!mania 4K World Cup 2017 - ¡Ya están abiertas las inscripciones!](https://osu.ppy.sh/home/news/2017-07-10-osumania-4k-world-cup-2017-registrations-now-open)
> - [Abierta la votación para el concurso de fanart de Mocha en Verano](https://osu.ppy.sh/home/news/2017-07-11-mocha-in-summer-fanart-contest-voting-open)
> - [osu!catch World Cup 2017 - Resultados](https://osu.ppy.sh/home/news/2017-07-14-osucatch-world-cup-2017-results)
> - [Programa de tutoría comunitaria otoño 2017 inscripciones abiertas](https://osu.ppy.sh/home/news/2017-07-18-community-mentorship-program-fall-2017-signups-now-open)

Reviewer: ```suggestion
- [osu!mania 4K World Cup 2017 - Registrations now open!](https://osu.ppy.sh/home/news/2017-07-10-osumania-4k-world-cup-2017-registrations-now-open)
- [Mocha in Summer Fanart Contest Voting Open](https://osu.ppy.sh/home/news/2017-07-11-mocha-in-summer-fanart-contest-voting-open)
- [osu!catch World Cup 2017 - Results](https://osu.ppy.sh/home/news/2017-07-14-osucatch-world-cup-2017-results)
- [Community Mentorship Program Fall 2017 Signups Now Open](https://osu.ppy.sh/home/news/2017-07-18-community-mentorship-program-fall-2017-signups-now-open)
```

## Agosto

> [Wisp X](https://osu.ppy.sh/beatmaps/artists/16) y [OISHII](https://osu.ppy.sh/beatmaps/artists/17) fueron anunciados como nuevos artistas destacados. Se abren las audiciones para osu!idol 2017. La osu! Coffee Hour se presentó como un pódcast. Se abrió una revisión para el nuevo aspecto de [Yuzu](/wiki/Mascots#yuzu) como un concurso de fanart.

Reviewer: ```suggestion
[Wisp X](https://osu.ppy.sh/beatmaps/artists/16) y [OISHII](https://osu.ppy.sh/beatmaps/artists/17) fueron anunciados como nuevos artistas destacados. Se abren las audiciones para osu!idol 2017. La osu! Coffee Hour se presentó como un pódcast. Se abrió una revisión para el nuevo aspecto de [Yuzu](/wiki/Mascots#yuzu) como un concurso de arte de fans.
```

> [Wisp X](https://osu.ppy.sh/beatmaps/artists/16) y [OISHII](https://osu.ppy.sh/beatmaps/artists/17) fueron anunciados como nuevos artistas destacados. Se abren las audiciones para osu!idol 2017. La osu! Coffee Hour se presentó como un pódcast. Se abrió una revisión para el nuevo aspecto de [Yuzu](/wiki/Mascots#yuzu) como un concurso de fanart.

Reviewer: ```suggestion
[Wisp X](https://osu.ppy.sh/beatmaps/artists/16) y [OISHII](https://osu.ppy.sh/beatmaps/artists/17) fueron anunciados como nuevos artistas destacados. Se abren las audiciones para osu!idol 2017. La osu! Coffee Hour se presentó como un pódcast. Se abrió una revisión para el nuevo aspecto de [Yuzu](/wiki/Mascots#yuzu) como un concurso de arte de fans.
```

Reviewer: and this?

> Links:

Reviewer: ```suggestion
Enlaces:
```

> - [Nuevo artista destacado: Wisp X](https://osu.ppy.sh/home/news/2017-08-10-new-featured-artist-wisp-x)
> - [Las audiciones para osu!idol 2017 ya están abiertas](https://osu.ppy.sh/home/news/2017-08-14-osu-idol-2017-auditions-now-open)
> - [Presentamos la osu! Coffee Hour](https://osu.ppy.sh/home/news/2017-08-15-introducing-the-osu-coffee-hour)
> - [Concurso Nuevo Look de Yuzu](https://osu.ppy.sh/home/news/2017-08-21-yuzus-new-look-contest)
> - [Nuevo artista destacado: OISHII](https://osu.ppy.sh/home/news/2017-08-30-new-featured-artist-oishii)

Reviewer: ```suggestion
- [w Featured Artist: Wisp X](https://osu.ppy.sh/home/news/2017-08-10-new-featured-artist-wisp-x)
- [osu!idol 2017 Auditions Now Open](https://osu.ppy.sh/home/news/2017-08-14-osu-idol-2017-auditions-now-open)
- [Introducing the osu! Coffee Hour](https://osu.ppy.sh/home/news/2017-08-15-introducing-the-osu-coffee-hour)
- [Yuzu's New Look Contest](https://osu.ppy.sh/home/news/2017-08-21-yuzus-new-look-contest)
- [New Featured Artist: OISHII](https://osu.ppy.sh/home/news/2017-08-30-new-featured-artist-oishii)
```

## Septiembre

> [\*namirin](https://osu.ppy.sh/beatmaps/artists/18) fue anunciada como otra artista destacada. Mapeo con recompensas regresa con recompensas vinculadas a canciones seleccionadas. La osu!mania 4K World Cup concluyó con Corea del Sur tomando el primer lugar de Estados Unidos. Se abrió la votación por el nuevo look de Yuzu.

Reviewer: ```suggestion
[\*namirin](https://osu.ppy.sh/beatmaps/artists/18) fue anunciada como otra artista destacada. Mapping with rewards regresa con recompensas vinculadas a canciones seleccionadas. La osu!mania 4K World Cup concluyó con Corea del Sur tomando el primer lugar de Estados Unidos. Se abrió la votación por el nuevo aspecto de Yuzu.
```

> Links:

Reviewer: ```suggestion
Enlaces:
```

> - [Nueva artista destacada: \*namirin](https://osu.ppy.sh/home/news/2017-09-10-new-featured-artist-namirin)
> - [Regresa mapeo con recompensas](https://osu.ppy.sh/home/news/2017-09-23-mapping-with-rewards-returns)
> - [osu!mania 4K World Cup - Resultados](https://osu.ppy.sh/home/news/2017-09-26-osu-mania-4k-world-cup-results)
> - [Ya está abierta la votación para el nuevo look de Yuzu](https://osu.ppy.sh/home/news/2017-09-29-yuzus-new-look-voting-now-open)

Reviewer: ```suggestion
- [New Featured Artist: \*namirin](https://osu.ppy.sh/home/news/2017-09-10-new-featured-artist-namirin)
- [Mapping With Rewards Returns](https://osu.ppy.sh/home/news/2017-09-23-mapping-with-rewards-returns)
- [osu!mania 4K World Cup - Results](https://osu.ppy.sh/home/news/2017-09-26-osu-mania-4k-world-cup-results)
- [Yuzu's New Look Voting Now Open](https://osu.ppy.sh/home/news/2017-09-29-yuzus-new-look-voting-now-open)
```

## Octubre

> Se abrió otro concurso de fanart de Halloween. Se abren las inscripciones para la osu! World Cup 2017. Unos días antes de Halloween, se abrió el concurso de fanart para Halloween. En Halloween, se revelaron los resultados del concurso de fanart.

Reviewer: ```suggestion
Se abrió otro concurso de arte de fans de Halloween. Se abren las inscripciones para la osu! World Cup 2017. Unos días antes de Halloween, se abrió el concurso de arte de fans para Halloween. En Halloween, se revelaron los resultados del concurso de arte de fans.
```

> Se abrió otro concurso de fanart de Halloween. Se abren las inscripciones para la osu! World Cup 2017. Unos días antes de Halloween, se abrió el concurso de fanart para Halloween. En Halloween, se revelaron los resultados del concurso de fanart.

Reviewer: ```suggestion
Se abrió otro concurso de arte de fans de Halloween. Se abren las inscripciones para la osu! World Cup 2017. Unos días antes de Halloween, se abrió el concurso de arte de fans para Halloween. En Halloween, se revelaron los resultados del concurso de arte de fans.
```

> Links:

Reviewer: ```suggestion
Enlaces:
```

> - [Concurso fanart de Halloween 2017](https://osu.ppy.sh/home/news/2017-10-09-halloween-fanart-contest)
> - [osu! World Cup 2017: ¡Ya están abiertas las inscripciones!](https://osu.ppy.sh/home/news/2017-10-13-osu-world-cup-2017-registrations-now-open)
> - [Abierta la votación para el Fanart de Halloween 2017](https://osu.ppy.sh/home/news/2017-10-24-halloween-2017-fanart-voting-now-open)
> - [Halloween desciende](https://osu.ppy.sh/home/news/2017-10-31-halloween-descends)

Reviewer: ```suggestion
- [Halloween 2017 Fanart Contest](https://osu.ppy.sh/home/news/2017-10-09-halloween-fanart-contest)
- [osu! World Cup 2017: Registrations now open!](https://osu.ppy.sh/home/news/2017-10-13-osu-world-cup-2017-registrations-now-open)
- [Halloween 2017 Fanart Voting Now Open](https://osu.ppy.sh/home/news/2017-10-24-halloween-2017-fanart-voting-now-open)
- [Halloween Descends](https://osu.ppy.sh/home/news/2017-10-31-halloween-descends)
```

## Noviembre

> [MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19) fue anunciado como otro artista destacado. peppy abrió un hilo en el foro para centralizar sus pensamientos sobre las pantallas táctiles; junto con algunas respuestas a las soluciones propuestas con frecuencia. El canal `#osu-performance` en el [servidor de Discord de osu!](/wiki/Community/osu!_Discord_server) se abrió para permitir una discusión abierta. Se abrió el concurso de Fanart de Invierno 2017.

Reviewer: ```suggestion
[MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19) fue anunciado como otro artista destacado. peppy abrió un hilo en el foro para centralizar sus pensamientos sobre las pantallas táctiles; junto con algunas respuestas a las soluciones propuestas con frecuencia. El canal `#osu-performance` en el [servidor de Discord osu!](/wiki/Community/osu!_Discord_server) se abrió para permitir una discusión abierta. Se abrió el Winter 2017 Fanart Contest.
```

> [MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19) fue anunciado como otro artista destacado. peppy abrió un hilo en el foro para centralizar sus pensamientos sobre las pantallas táctiles; junto con algunas respuestas a las soluciones propuestas con frecuencia. El canal `#osu-performance` en el [servidor de Discord de osu!](/wiki/Community/osu!_Discord_server) se abrió para permitir una discusión abierta. Se abrió el Winter 2017 Fanart Contest.

Reviewer: ```suggestion
[MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19) fue anunciado como otro artista destacado. peppy abrió un hilo en el foro para centralizar sus pensamientos sobre las pantallas táctiles; junto con algunas respuestas a las soluciones propuestas con frecuencia. El canal `#osu-performance` en el [servidor de Discord osu!](/wiki/Community/osu!_Discord_server) se abrió para permitir una discusión abierta. Se abrió el Winter 2017 Fanart Contest.
```

> [MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19) fue anunciado como otro artista destacado. peppy abrió un hilo en el foro para centralizar sus pensamientos sobre las pantallas táctiles; junto con algunas respuestas a las soluciones propuestas con frecuencia. El canal `#osu-performance` en el servidor de Discord de osu!dev se abrió para permitir una discusión abierta. Se abrió el concurso de Fanart de Invierno 2017.

Reviewer: ```suggestion
[MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19) fue anunciado como otro artista destacado. peppy abrió un hilo en el foro para centralizar sus pensamientos sobre las pantallas táctiles; junto con algunas respuestas a las soluciones propuestas con frecuencia. El canal `#osu-performance` en el servidor de Discord osu!dev se abrió para permitir una discusión abierta. Se abrió el concurso de arte de fans de invierno de 2017.
```

> Links:

Reviewer: ```suggestion
Enlaces:
```

> - [Nuevo artista destacado: MOtOLOiD](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid)
> - [osu!, pantallas táctiles y tú](https://osu.ppy.sh/community/forums/topics/665986)
> - [Concurso de Fanart de Invierno 2017](https://osu.ppy.sh/home/news/2017-11-26-winter-2017-fanart-contest)

Reviewer: ```suggestion
- [New Featured Artist: MOtOLOiD](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid)
- [osu!, touchscreens and you](https://osu.ppy.sh/community/forums/topics/665986)
- [Winter 2017 Fanart Contest](https://osu.ppy.sh/home/news/2017-11-26-winter-2017-fanart-contest)
```

## Diciembre

> Se publicaron los resultados del concurso de fanart de invierno de 2017, premiando a [Leissss](https://osu.ppy.sh/users/4750716) con la mayor cantidad de votos. La OWC 2017 concluyó con la victoria de Polonia. Gracias a LiquidPL, la vergonzoza página de descarga de la osu!web se rehizo. El día antes de Navidad, se anunció otro puñado de contribuidores de la comunidad. Todos los jugadores recibieron el estado de osu!supporter temporalmente con fines de prueba, lo que casualmente sucedió en Nochebuena. peppy finaliza el año con un video, dando a 2017 una revisión del año con respecto a la versión [lazer](/wiki/Client/Release_stream/Lazer) de osu!.

Reviewer: ```suggestion
Se publicaron los resultados del Winter 2017 Fanart Contest, premiando a [Leissss](https://osu.ppy.sh/users/4750716) con la mayor cantidad de votos. La OWC 2017 concluyó con la victoria de Polonia. Gracias a LiquidPL, la vergonzoza página de descarga de la osu!web se rehizo. El día antes de Navidad, se anunció otro puñado de contribuidores de la comunidad. Todos los jugadores recibieron el estado de osu!supporter temporalmente con fines de prueba, lo que casualmente sucedió en Nochebuena. peppy finaliza el año con un video, dando a 2017 una revisión del año con respecto a la versión [lazer](/wiki/Client/Release_stream/Lazer) de osu!.
```

> Se publicaron los resultados del concurso de fanart de invierno de 2017, premiando a [Leissss](https://osu.ppy.sh/users/4750716) con la mayor cantidad de votos. La OWC 2017 concluyó con la victoria de Polonia. Gracias a LiquidPL, la vergonzoza página de descarga de la osu!web se rehizo. El día antes de Navidad, se anunció otro puñado de colaboradores de la comunidad. Todos los jugadores recibieron el estado de osu!supporter temporalmente con fines de prueba, lo que casualmente sucedió en Nochebuena. peppy finaliza el año con un video, dando a 2017 una revisión del año con respecto a la versión [lazer](/wiki/Client/Release_stream/Lazer) de osu!.

Reviewer: ```suggestion
Se publicaron los resultados del concurso de arte de fans de invierno de 2017, premiando a [Leissss](https://osu.ppy.sh/users/4750716) con la mayor cantidad de votos. La OWC 2017 concluyó con la victoria de Polonia. Gracias a LiquidPL, la vergonzoza página de descarga de la osu!web se rehizo. El día antes de Navidad, se anunció otro puñado de contribuidores de la comunidad. Todos los jugadores recibieron el estado de osu!supporter temporalmente con fines de prueba, lo que casualmente sucedió en Nochebuena. peppy finaliza el año con un vídeo, dando a 2017 una revisión del año con respecto a la versión [lazer](/wiki/Client/Release_stream/Lazer) de osu!.
```

> Links:

Reviewer: ```suggestion
Enlaces:
```

> - [¡El invierno esta aqui!](https://osu.ppy.sh/home/news/2017-12-14-winter-is-here)
> - [Concluye la osu! World Cup 2017](https://osu.ppy.sh/home/news/2017-12-21-owc-2017-concludes)
> - [Colaboradores de la comunidad: 2017](https://osu.ppy.sh/home/news/2017-12-24-community-contributors-2017)
> - [osu!status en Twitter](https://twitter.com/osustatus/status/945098179183181824)
> - [¡y así termina el 2017!](https://www.youtube.com/watch?v=5x7VnC1R0Do)

Reviewer: ```suggestion
- [Winter is Here!](https://osu.ppy.sh/home/news/2017-12-14-winter-is-here)
- [osu! World Cup 2017 Concludes](https://osu.ppy.sh/home/news/2017-12-21-owc-2017-concludes)
- [Community Contributors: 2017](https://osu.ppy.sh/home/news/2017-12-24-community-contributors-2017)
- [osu!status on Twitter](https://twitter.com/osustatus/status/945098179183181824)
- [and so 2017 comes to an end!](https://www.youtube.com/watch?v=5x7VnC1R0Do)
```
